### ISSUE TITLE
When the "package packagename" is missing in the proto file, an error occurs during the invocation.

### ISSUE THREAD

**Issue by xiaoyuuuuuupeng** [apache/dubbo#13372]
- [ ] I have searched the [issues](https://github.com/apache/dubbo/issues) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: 3.2.7
* Operating System version: macos 13.2.1 (22D68)
* Java version: 1.8

### Steps to reproduce this issue

1. proto文件
```proto
syntax = "proto3";

option java_multiple_files = true;
option java_package = "org.example.protogen";
option java_outer_classname = "PredictorProto";


message PredictRequest {
  string model = 1;
  string imei = 2;
  repeated string itemId = 3;
}

message PredictResponse {
  string errMsg = 1;
  string model = 2;
  string predictionMap = 3;
}

service PredictService {
  rpc Predict (PredictRequest) returns (PredictResponse) {}
}
```
2. generate proto files and build a trip server 
3. build a native grpc client

### Expected Behavior
success

### Actual Behavior
failed 

If there is an exception, please attach the exception trace:
- native grpc client side:
![image](https://github.com/apache/dubbo/assets/30964063/18a1e2d9-201e-4ec8-8ce1-92f2f39cf5bf)

 
```
Exception in thread "main" io.grpc.StatusRuntimeException: UNIMPLEMENTED: Service not found:PredictService
	at io.grpc.stub.ClientCalls.toStatusRuntimeException(ClientCalls.java:271)
	at io.grpc.stub.ClientCalls.getUnchecked(ClientCalls.java:252)
	at io.grpc.stub.ClientCalls.blockingUnaryCall(ClientCalls.java:165)
	at org.example.protogen.PredictServiceGrpc$PredictServiceBlockingStub.predict(PredictServiceGrpc.java:157)
	at com.example.userinfoservice.Main2Application.main(Main2Application.java:23)
```

**Comment by xiaoyuuuuuupeng** [comment]
原因是在没```package xxx;```时，会导致Dubbo3TripleGenerator生成的Interface中 SERVICE_NAME错误
![image](https://github.com/apache/dubbo/assets/30964063/21c76a93-714d-447c-9c50-b9b18ee4d88d)

**Comment by xiaoyuuuuuupeng** [comment]
Will be fixed in 3.3
